db/schema.rb

```diff
@@ -57,6 +57,9 @@
     t.integer "zip"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "status", default: "pending"
+    t.bigint "user_id"
```

Comment:
This is the kind of defaulting I was looking for last night but it would not work when I did it to set default users.

Reply:
its a real bitch.  REAL BITCH.  internet was not very helpful, and its actaully solved by making a seperate migration